kernel/configs/x86_64_kata_kvm_4.14.x

```diff
@@ -138,8 +138,9 @@ CONFIG_RCU_NEED_SEGCBLIST=y
 CONFIG_CONTEXT_TRACKING=y
 # CONFIG_CONTEXT_TRACKING_FORCE is not set
 CONFIG_RCU_NOCB_CPU=y
-# CONFIG_BUILD_BIN2C is not set
-# CONFIG_IKCONFIG is not set
+CONFIG_BUILD_BIN2C=y
+CONFIG_IKCONFIG=y
```

Comment:
You have enabled the above options. If that was the intention, these should go in a separate commit with an explanation for the options.

Reply:
seems that was enabled accidentally after the oldconfig, removed 👍